Misc/NEWS.d/next/Library/2019-09-10-10-59-50.bpo-37251.8zn2o3.rst

```diff
@@ -0,0 +1,3 @@
+Removes `__code__` check in AsyncMock's `_is_async_obj` that incorrectly
```

Comment:
Nit: I think we can remove `_is_async_obj` since it's just a private function.

```suggestion
Remove `__code__` check in AsyncMock that incorrectly
```